cloud_testing/kotlin_poc/src/main/kotlin/ftl/gc/GcStorage.kt

```diff
@@ -23,22 +23,29 @@ object GcStorage {
         storageOptions.service
     }
 
-    private fun upload(apk: String, rootGcsBucket: String, runGcsPath: String): String {
-        val apkFileName = Paths.get(apk).fileName.toString()
-        val gcsApkPath = GCS_PREFIX + join(rootGcsBucket, runGcsPath, apkFileName)
+    private fun upload(file: String, rootGcsBucket: String, runGcsPath: String): String {
+        return upload(file = file,
+                fileBytes = Files.readAllBytes(Paths.get(file)),
+                rootGcsBucket = rootGcsBucket,
+                runGcsPath = runGcsPath)
+    }
+
+    private fun upload(file: String, fileBytes: ByteArray, rootGcsBucket: String, runGcsPath: String): String {
```

Comment:
I wonder if we should add some kind of android/ios in the folder path. If users are running android and iOS testing, we will see both builds together, right?

Reply:
If you set the root bucket, then android and iOS will go in different folders. For example `rootGcsBucket: earlgrey-swift`